Makefile

```diff
@@ -0,0 +1,63 @@
+NAMESPACE := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["namespace"])')
+NAME := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["name"])')
+VERSION := $(shell python -c 'import yaml; print(yaml.safe_load(open("galaxy.yml"))["version"])')
+MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.json
+
+ROLES := $(wildcard roles/*)
+DOCS := $(wildcard docs/*) $(wildcard docs/*/*)
+META := $(wildcard meta/*)
+PLAYBOOKS := $(wildcard playbooks/*) $(wildcard playbooks/*/*) $(wildcard playbooks/*/*/*)
+METADATA := galaxy.yml COPYRIGHT LICENSE README.md requirements.yml
+DEPENDENCIES := $(METADATA) $(foreach ROLE,$(ROLES),$(wildcard $(ROLE)/*/*)) $(META) $(DOCS) $(PLAYBOOKS)
+
+TOX_ENV ?= py37-ansible2.9-release-static
+PYTHON_VERSION = $(shell python -c 'import sys; print("{}.{}".format(sys.version_info.major, sys.version_info.minor))')
+SANITY_OPTS =
+
+default: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help           to show this message"
+	@echo "  info           to show infos about the collection"
+	@echo "  lint           to run code linting"
+	@echo "  sanity         to run santy tests"
+	@echo "  test           to run molecule tests"
+	@echo "  dist           to build the collection artifact"
+
+info:
+	@echo "Building collection $(NAMESPACE)-$(NAME)-$(VERSION)"
+	@echo "  roles:\n $(foreach ROLE,$(notdir $(ROLES)),   - $(ROLE)\n)"
+
+lint: $(MANIFEST)
+	yamllint -f parsable roles
+	ansible-playbook --syntax-check roles/*.yaml | grep -v '^$$'
```

Comment:
If this isn't the kind of linting done by the CI, i would remove it.

Also if the CI would call into this Makefile for (almost) all operations, this would help debugging issues tremendously.
(Maybe not in _this_ PR though)